Internal/interface/postgres/linkrepo/linkrepo.go

```diff
@@ -169,3 +171,18 @@ func (p *Postgres) ChangeLink(oldLink string, newLink string, accountId string)
 	}
 	return nil
 }
+
+const querySetBad = `
+	UPDATE oldLinkByNewLink
+	SET isBad = FALSE
```

Comment:
Not too bad to present some valid state using inverted logic: - "Is A bad?" - "Nope!" Could you think of some other way of formalizing the concept of the URL's _current_ status?